### ISSUE TITLE
compat with trinkets slot 

### ISSUE THREAD

**Issue by DadoIrie** [Andrew6rant/Thanks--I-fixed-your-Inventory#2]
Can TIFYI tap into [Trinkets](https://modrinth.com/mod/trinkets) slots? Giving it optional compatibility if trinkets is present?
![image](https://github.com/Andrew6rant/Thanks--I-fixed-your-Inventory/assets/61109153/93c795cf-4d48-44cf-800c-925a3abbb2b4)
basically only the 3 red marked ones - the left outer ones ain't needed

EDIT: don't worry about the offhand/shield thingy - it's just me using the inventory.png from TIFYI with vanilla slot configs

**Comment by Andrew6rant** [comment]
Yeah, I'll look into supporting Trinkets' slots in TIFYI. Trinkets doesn't have a way to change slot locations in their API so I have to write some really gross code to hook into it.

I've actually done it before, for another project of mine, LittleBigInv: https://github.com/Andrew6rant/BigInv. (see how the minecart slot and disabled slots are aligned with the offhand icon)

![LittleBigInv Showcase](https://github.com/Andrew6rant/Thanks--I-fixed-your-Inventory/assets/57331134/8198b737-5525-48c0-b156-1cd8e92cdf97)

That mod hard depends on Trinkets, so I can directly modify its code. I don't want to do that for this mod, so I also need to make a mixin plugin.

**Comment by Andrew6rant** [comment]
TIFYI will not move the trinkets slots on the left to the right, that can be done through datapacks with just Trinkets API, I'm pretty sure.

**Comment by Andrew6rant** [comment]
I haven't started on Trinkets compat (because I need to make a mixin plugin for it), but I've implemented configs in 1.1.0 to move the recipe book and "crafting" text (also added a config to change the size of the player inventory render)

**Comment by jptrzy** [comment]
I haven't updated it for a while, but [this mod](https://modrinth.com/mod/trinkets-curios-theme) could be relevant.
